package.json

```diff
@@ -156,9 +156,6 @@
     "webpack": "^5.95.0",
     "xml-formatter": "^3.6.3"
   },
-  "overrides": {
```

Comment:
@ValenberghsSven Door deze override kreeg ik zowel lokaal als binnen te testen (maar niet altijd om één of andere reden) foutmeldingen ivm importen van modules.

Dit was gerelateerd aan de opmerking van @windvis hier (onderaan): https://github.com/appuniversum/ember-appuniversum/releases/tag/v3.7.0.

Als ik deze verwijder krijg ik lokaal niet direct errors, maar ik laat zeker ook even alle tests lopen. Laat je weten of dit ok is?

Reply:
Ik vermoed wel ok.
Die override was er ooit gekomen omdat die specifieke patch onze builds stuk deed.
Die overriden was de snelste oplossing.

Vermoedelijk is het issue dat we ermee hadden gefixt geweest in de volgende patch
https://github.com/emberjs/babel-plugin-ember-template-compilation/releases/tag/v2.2.3-babel-plugin-ember-template-compilation

We hebben dit gewoon niet meer herbekeken.

Reply:
zolang we die error niet krijgen in jenkins mag die override wel weg denk ik.